makefont/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use std::env::args;
+use std::fs::write;
+use std::process::exit;
+use image::GenericImageView;
+
+const CHARACTER_WIDTH: u32 = 9;
```

Comment:
What would you say about changing the font image format? A 9-wide grid is impractical, and the first row thing is really unwieldy.

I'm thinking, instead, of this:
- Read pure black and pure white pixels as a character, for any amount of pixels;
- Once encountering a pixel that is neither, bail if it is not the "null colour" (or remember it as such if no null colour has been set yet)
- Ignore all pixels until the next that isn't a null colour
- Additionally, skip any line (between character lines) that is only comprised of null pixels

This would allow people to align characters however they see fit (including using whatever grid size they prefer), while removing the aforementioned quirkiness.

What do you think?